Introduce failsafe for integrationtest for test  results. (#252)
cloudogu/gitops-playground: docs/developers.md

## Testing

> 1. There are end to end tests implement by Junit. Classes marked with 'IT' and the end.

Reviewer: ```suggestion
1. There are integration tests implemented by Junit. Classes marked with 'IT' and the end.
```

> 2. Long living tests marked with 'LongIt'.

Reviewer: ```suggestion
2. Long running tests are marked with 'LongIT'.
```

3. Main Branch executes both, feature-branches only IT.

### Usage

> Runnable seperatly via maven.

Reviewer: ```suggestion
Runnable separately via maven.
```

``

> failsafe:integration-test -f pom.xml

Reviewer: ```suggestion
mvn failsafe:integration-test -f pom.xml
```

``
To run long living test, use maven with profile: long-running 
``

> failsafe:integration-test -f pom.xml -P long-running

Reviewer: ```suggestion
mvn failsafe:integration-test -f pom.xml -P long-running
```
